### DISCUSSION TITLE
Skip Connections

### DISCUSSION THREAD

**Discussion by jomar0** [BindsNET/bindsnet#681] [Q&A]
Hello, sorry if this is a silly question,
How would I create skip connections in this framework?
Thanks

**Comment by Hananel-Hazan** [comment]
Simply don't define a connection between the layers.
If you need to define some connections to skip, you can use a mask in the run function. This mask should consist of boolean values: one to enable the connection (1) and zero (0) for connections that don't need to hold weights.
It's worth noting that because these options exist within the run function, it's possible to use them dynamically or as part of a learning scheme that opens and closes connections on the fly.
